specification/eventhub/resource-manager/Microsoft.EventHub/stable/2017-04-01/EventHub.json

```diff
@@ -2074,6 +2074,54 @@
           "nextLinkName": "nextLink"
         }
       }
+    },
+    "/subscriptions/{subscriptionId}/providers/Microsoft.EventHub/locations/{locationName}/namespaceOperations/{operationResultId}": {
```

Comment:
It looks like you're trying to expose the operation status of an async api. There is a pattern in ARM for this, you'd expose:

/subscriptions/{subscriptionId}/providers/Microsoft.EventHub/locations/{locationName}/operationsStatus
https://github.com/Azure/azure-resource-manager-rpc/blob/master/v1.0/Addendum.md#operation-resource-format

Reply:
@rrama-Microsoft can you please take look here